site/global-guides/voice/authentication.mdx

```diff
@@ -0,0 +1,75 @@
+---
+id: inAppAuthGuide
+title: Authentication for Global In-App Calling
+slug: /voice/guides/inAppCalling/authentication
+description: Authentication In-App Calling (WebRTC)
+keywords:
+  - bandwidth
+  - voice
+hide_title: false
+image: '@site/static/img/bw-icon.svg'
+---
+## Retrieving an access token
+
+:::caution
+
+The token should be retrieved from a server running in a secure environment and securely provided to clients. Client-side javascript does not have a mechanism for hiding these credentials so DO NOT place these directly in your client-side code.
+
+Bandwidth accepts no responsibility for the loss of account credentials and any resulting network traffic, fraud, or undesired account access that results from failing to manage account access credentials in a completely secure manner.
+:::
+
+
+The process to retrieve an access token you require you to have your ClientID and ClientSecret which you generated in the UI. You will then make a call to our token endpoint with these credentials. We will then return to you an `access_token`, `id_token`, `token_type`, and `expires_in` (this is the number of seconds before the token will expire).
+
+```mermaid
+sequenceDiagram
+    Application->>BW Global API AuthServer: GET /api/v1/oauth2/token?scope=voice.webrtc <br> Headers: { Authorization: Basic PEJhc2ljIEF1dGggVXNlcm5hbWU+OjxCYXNpYyBBdXRoIFBhc3N3b3JkPg== }
+    BW Global API AuthServer->>BW Global API AuthServer: Validate credentials
+    BW Global API AuthServer->>Application: 200 OK <br> {"access_token": "...", "access_token_id": "...", "token_type": "Bearer", "expires_in": 600}
+    Application->>WebRTC Gateway: GET /someresource <br> Headers: {Authorization: <token_type> <access_token>}
+```
+### Request
+In order to retrieve the token you will use Basic Auth `ClientID:ClientSecret`. Below is a sample request
+
+```sh
+curl --request POST \
+  --url https://<voxbones_token_providerd_url>/api/v1/oauth2/token?scope=voice.webrtc \
+  --header 'Authorization: Basic PEJhc2ljIEF1dGggVXNlcm5hbWU+OjxCYXNpYyBBdXRoIFBhc3N3b3JkPg==' \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data grant_type=client_credentials
+```
+Now that we've shown the request let's break down the parts of the call.
```

Comment:
```suggestion
Now that we've shown the request let's break down the parts of the API call.
```
Because we are telecom, we have voice calls and I usually like to make the distinction of an API call because it's not always so. "Call" can mean a voice call.